src/0x.ts

```diff
@@ -162,15 +162,16 @@ export class ZeroEx {
      * Instantiates a new ZeroEx instance that provides the public interface to the 0x.js library.
      * @param   provider    The Web3.js Provider instance you would like the 0x.js library to use for interacting with
      *                      the Ethereum network.
+     * @param   gasPrice    The gas price to use for sending transactions. Defaults to 21 GWei.
      * @return  An instance of the 0x.js ZeroEx class.
      */
-    constructor(provider: Web3Provider) {
+    constructor(provider: Web3Provider, gasPrice?: BigNumber.BigNumber) {
```

Comment:
So I really do feel strongly that we should make this a config obj called `configs`. I am 100% sure this won't be the last config param we will add to this library and it'll be cleaner to write:

```
const configs = {
  gasPrice,
  requestTimeoutLimit,
  verbose,
};
const zeroEx = new ZeroEx(provider, configs);
```

Then:

```
const zeroEx = new ZeroEx(provider, gasPrice, requestTimeoutLimit, verbose);
```

Especially as the number of configs grows.